### PR TITLE
Translated from Tamil to English

### PR DESCRIPTION
**Pull request by VasuShrivastava**
Translated from Tamil to English so that the game could be accessed by more people.

### CODE REVIEW

File: mobile.html [hunk 1]
```diff
@@ -49,14 +49,14 @@
 
             <div class="collapse navbar-collapse" id="navbarToggler">
                 <ul class="navbar-nav mr-auto mt-2 mt-lg-0">
-                    <li class="nav-item"><a class="nav-link" href=""><i class="pr-1 fa fa-home"></i>முகப்பு</a></li>
-                    <li class="nav-item"><a class="nav-link" href=""><i class="pr-1 fa fa-globe"></i>எங்களை பற்றி</a>
+                    <li class="nav-item"><a class="nav-link" href=""><i class="pr-1 fa fa-home"></i>Interface</a></li>
```
Comment (MaathavanJkr) [inline mobile.html:52]: Interface -> Home

File: mobile.html [hunk 2]
```diff
@@ -49,14 +49,14 @@
...
+                    <li class="nav-item"><a class="nav-link" href=""><i class="pr-1 fa fa-globe"></i>About us</a>
                     </li>
                     <li class="nav-item active"><a class="nav-link" href="mobile.html"><i
-                                class="pr-1 fa fa-flag"></i>போட்டி</a></li>
-                    <li class="nav-item"><a class="nav-link" href="rank.html"><i class="pr-1 fa fa-list-ol"></i>நிலை</a>
+                                class="pr-1 fa fa-flag"></i>Competition</a></li>
+                    <li class="nav-item"><a class="nav-link" href="rank.html"><i class="pr-1 fa fa-list-ol"></i>Condition</a>
```
Comment (MaathavanJkr) [inline mobile.html:57]: It is directing to rank.html which shows ranks. Condition -> Rank

File: mobile.html [hunk 3]
```diff
@@ -81,14 +81,14 @@
...
-                <div class="text-center" id="answer">விடைகள்: 0/10</div>
+                <div class="text-center" id="answer">Answers: 0/10</div>
             </div>
             <div id="myModal" class="modal">
 
                 <!-- Modal content -->
                 <div class="modal-content">
                     <div class="modal-header">
-                        <div id="modal-header-text">வரைபடவி</div>
+                        <div id="modal-header-text">வரைபடவி-Map</div>
```
Comment (MaathavanJkr) [inline mobile.html:91]: It's is the name of the App. Just use Varaipadavi

File: mobile.html [hunk 4]
```diff
@@ -109,13 +109,12 @@
...
                         </div>
                         <div id="modal-text"></div>
-                        <button class="vp-btn" id="start-btn" onclick="startGame()">தொடங்குக</button>
-                        <a href="study.html"><button class="vp-btn" id="study-link-btn">பயில்க</button></a>
-                        <input type="text" id="username-input" placeholder="உங்கள் பெயரை உள்ளிடுக" />
-                        <button class="vp-btn" id="add-time-btn" onclick="addRankToDatabase()">புள்ளியை
-                            சேர்</button>
-                        <button class="vp-btn" id="replay-btn" onclick="restartGame()">திரும்ப விளையாடு</button>
-                        <a href="rank.html"><button class="vp-btn" id="rank-link-btn">நிலைகளை காட்டு</button></a>
+                        <button class="vp-btn" id="start-btn" onclick="startGame()">Get Started</button>
```
Comment (MaathavanJkr) [inline mobile.html:112]: Get Started -> Start

File: mobile.html [hunk 5]
```diff
@@ -109,13 +109,12 @@
...
+                        <a href="study.html"><button class="vp-btn" id="study-link-btn">Learn</button></a>
+                        <input type="text" id="username-input" placeholder="Enter your name" />
+                        <button class="vp-btn" id="add-time-btn" onclick="addRankToDatabase()">Add the point</button>
```
Comment (MaathavanJkr) [inline mobile.html:115]: Add Score

File: mobile.html [hunk 6]
```diff
@@ -109,13 +109,12 @@
...
+                        <button class="vp-btn" id="replay-btn" onclick="restartGame()">Play again</button>
+                        <a href="rank.html"><button class="vp-btn" id="rank-link-btn">Show Levels</button></a>
```
Comment (MaathavanJkr) [inline mobile.html:117]: Should be just Rank

File: mobile.html [hunk 7]
```diff
@@ -189,19 +188,19 @@
...
             if ($("#" + place).length == 0) {
-                showSnackbar("தவறு, அது " + place.replace("_", " "), "red", 2000);
+                showSnackbar("Wrong, that is " + place.replace("_", " "), "red", 2000);
             } else {
                 answers += 1;
                 if (answers >= 10) {
                     stopwatch.stop();
                     document.getElementById('question').innerHTML = '';
-                    showModal('வாழ்த்துக்கள்!', 'உங்கள் புள்ளி நேரம்: <b>' + stopwatch.lap().replace(/;/g, "a") + '</b>');
+                    showModal('Congratulations!', 'Your point time: <b>' + stopwatch.lap().replace(/;/g, "a") + '</b>');
```
Comment (MaathavanJkr) [inline mobile.html:197]: Your point time -> Your Score is

File: mobile.html [hunk 8]
```diff
@@ -244,7 +243,7 @@
             })
             $("#add-time-btn").hide();
             $("#username-input").hide();
-            showModal('வெற்றி!', 'உங்கள் புள்ளி சேர்க்கப்பட்டது, ' + name + ".\n மறுதடவை விளையாடப் போகின்றீர்களா?");
+            showModal('Success!', 'Your point is Added, ' + name + ".\n Are you going to play again?");
```
Comment (MaathavanJkr) [inline mobile.html:246]: Use score instead of point

File: study.html [hunk 9]
```diff
@@ -77,16 +77,16 @@
             </map>
 
             <div id="study-area">
-                <div>காட்ட வேண்டிய இடங்களை தெரிவு செய்க</div>
-                <button class="vp-btn" id="open-btn" onclick="modal.style.display = 'block';">தெரிக</button>
+                <div>Choose places to show</div>
+                <button class="vp-btn" id="open-btn" onclick="modal.style.display = 'block';">Find out</button>
```
Comment (MaathavanJkr) [inline study.html:81]: Find out -> select

**Review by MaathavanJkr — CHANGES_REQUESTED** [review]

**Comment by MaathavanJkr** [comment]
Nice work. And you have used a translator ig. Anyways consider the above changes. ;)

**Review by VasuShrivastava — COMMENTED** [review]
Reviewed all the changes. Sorry for the trouble

File: mobile.html [hunk 10]
```diff
@@ -189,19 +188,19 @@
...
                     document.getElementById('question').innerHTML = '';
-                    showModal('வாழ்த்துக்கள்!', 'உங்கள் புள்ளி நேரம்: <b>' + stopwatch.lap().replace(/;/g, "a") + '</b>');
+                    showModal('Congratulations!', 'Your Score time: <b>' + stopwatch.lap().replace(/;/g, "a") + '</b>');
                     $("#username-input").show();
                     $("#replay-btn").show();
                     $("#add-time-btn").show();
                 } else {
-                    showSnackbar("சரியான பதில்.", "green", 800);
-                    $("#answer").replaceWith('<div class="text-center" id="answer">விடைகள்: ' + answers + '/10</div>');
+                    showSnackbar("The correct answer.", "green", 800);
```
Comment (MaathavanJkr) [inline mobile.html:202]: There should not be a "The"

File: mobile.html [hunk 11]
```diff
@@ -244,7 +243,7 @@
             })
             $("#add-time-btn").hide();
             $("#username-input").hide();
-            showModal('வெற்றி!', 'உங்கள் புள்ளி சேர்க்கப்பட்டது, ' + name + ".\n மறுதடவை விளையாடப் போகின்றீர்களா?");
+            showModal('Success!', 'Your Score is, ' + name + ".\n Are you going to play again?");
```
Comment (MaathavanJkr) [inline mobile.html:246]: Your Score is -> You score has been added

File: web.html [hunk 12]
```diff
@@ -251,7 +251,7 @@
             })
             $("#add-time-btn").hide();
             $("#username-input").hide();
-            showModal('வெற்றி!', 'உங்கள் புள்ளி சேர்க்கப்பட்டது, ' + name + ".\n மறுதடவை விளையாடப் போகின்றீர்களா?");
+            showModal('Success!', 'Your Score is Added , ' + name + ".\n Are you going to play again?");
```
Comment (MaathavanJkr) [inline web.html:254]: here too

**Review by MaathavanJkr — COMMENTED** [review]
Good work. Just some small changes. I will merge after this. :)

File: mobile.html [hunk 13]
```diff
@@ -49,14 +49,14 @@
...
                 <ul class="navbar-nav mr-auto mt-2 mt-lg-0">
-                    <li class="nav-item"><a class="nav-link" href=""><i class="pr-1 fa fa-home"></i>முகப்பு</a></li>
-                    <li class="nav-item"><a class="nav-link" href=""><i class="pr-1 fa fa-globe"></i>எங்களை பற்றி</a>
+                    <li class="nav-item"><a class="nav-link" href=""><i class="pr-1 fa fa-home"></i>Home</a></li>
+                    <li class="nav-item"><a class="nav-link" href=""><i class="pr-1 fa fa-globe"></i>About us</a>
                     </li>
                     <li class="nav-item active"><a class="nav-link" href="mobile.html"><i
-                                class="pr-1 fa fa-flag"></i>போட்டி</a></li>
-                    <li class="nav-item"><a class="nav-link" href="rank.html"><i class="pr-1 fa fa-list-ol"></i>நிலை</a>
+                                class="pr-1 fa fa-flag"></i>Competition</a></li>
```
Comment (MaathavanJkr) [inline mobile.html:56]: Use Quiz instead of competition

File: mobile.html [hunk 14]
```diff
@@ -81,14 +81,14 @@
...
-                <div class="text-center" id="answer">விடைகள்: 0/10</div>
+                <div class="text-center" id="answer">Answers: 0/10</div>
             </div>
             <div id="myModal" class="modal">
 
                 <!-- Modal content -->
                 <div class="modal-content">
                     <div class="modal-header">
-                        <div id="modal-header-text">வரைபடவி</div>
+                        <div id="modal-header-text">வரைபடவி-Varaipadavi</div>
```
Comment (MaathavanJkr) [inline mobile.html:91]: வரைபடவி-Varaipadavi this is long. just use Varaipadavi

**Review by VasuShrivastava — COMMENTED** [review]
Reviewed all the changes.

**Comment by MaathavanJkr** [comment]
Good work. Thank you! :)

**Pull request merged**